.github/workflows/dotnet-desktop.yml

```diff
@@ -28,6 +28,17 @@ jobs:
           } else {
             Write-Error "python node dll was not found!"
           }
-      - name: Test with the dotnet CLI
-        run: dotnet test ${{ github.workspace }}\DSIronPython  
-
+      - name: Run test with the dotnet CLI
+        run: |
+          dotnet test ${{ github.workspace }}\DSIronPython --logger trx --results-directory ${{ github.workspace }}\DSIronPython\TestResults
+      - name: Publish test results
+        uses: bibipkins/dotnet-test-reporter@v1.3.3
+        with:
+          github-token: ${{ github.token }}
+          results-path: ${{ github.workspace }}/DSIronPython/TestResults/*.trx
+      - name: Upload build artifact
+        uses: actions/upload-artifact@v3.1.3
```

Comment:
will this also save PR builds? Do we want that?

Reply:
My intention for uploading package artifact to use it for testing so the retention is only 7 days. I think having artifact from a pr can also be useful for testing.